packages/local-build-plugin/src/exit.ts

```diff
@@ -4,7 +4,12 @@ const handlers: (() => void | Promise<void>)[] = [];
 let shouldExitStatus = false;
 
 export function listenForInterrupts(): void {
+  let handlerCalled = false;
   const handleExit = async (): Promise<void> => {
+    if (handlerCalled) {
```

Comment:
not sure why, but if I call `childProcess.kill()` local build receives to signals